classic/form/field/GeocoderComboBox.js

```diff
@@ -296,6 +296,7 @@ Ext.define('GeoExt.form.field.GeocoderComboBox', {
     unRestrictExtent: function() {
         var me = this;
         // unbinding moveend event
+        ol.Observable.unByKey(me.updateExtraParamsListenerKey_);
         me.map.un('moveend', me.updateExtraParams, me);
```

Comment:
this will not work. I think the `me.updateExtraParamsListenerKey_` should get set and this line should get removed